test/test_one_dimensional_shallow_water_model.jl

```diff
@@ -0,0 +1,49 @@
+# # One dimensional shallow water example
+
+using Oceananigans, Oceananigans.Advection
+
+using Oceananigans.Models # should import the shallow water model stuff
+#include("../src/Models/ShallowWaterModels/ShallowWaterModels.jl")
+using .ShallowWaterModels: ShallowWaterModel
+
+grid = RegularCartesianGrid(size=(64, 1, 1), extent=(2π, 2π, 2π))
+
+model = ShallowWaterModel(        grid = grid,
+                          architecture = CPU(),
+                             advection = nothing, 
+                              coriolis = nothing, 
+                             solution = nothing,
+                              tracers = nothing#,
+#                          timestepper = RungeKutta3 
+                                  )
+
+width = 0.3
+ h(x, y, z)  = 1.0 + 0.1 * exp(-(x - π)^2 / (2width^2)); 
+uh(x, y, z) = 0.0
+vh(x, y, z) = 0.0 
+
+include("../src/Models/ShallowWaterModels/set_shallow_water_model.jl")
+set!(model, uh = uh, vh = vh, h = h)
+
+using Plots
+using Oceananigans.Grids: xnodes 
+
+x = xnodes(model.solution.h)
+
+h_plot = plot(x, interior(model.solution.h)[:, 1, 1],
+              linewidth = 3,
+                  label = "t = 0",
+                 xlabel = "x (m)",
+                 ylabel = "",
+                 title  = "Height (m)")
+
+display(h_plot)
+savefig("initial_height.png")
+
+simulation = Simulation(model, Δt = 0.1, stop_iteration = 10)
```

Comment:
We can't plot a solution in a test. I think the test we need here is something simpler: instantiate a `ShallowWaterModel`, wrap it in `simulation`, and then call `run!(simulation)`. The test should pass if the model can be time-stepped without throwing an error.

We tested `set!` in a previous test, so we don't need to use that here.